### PR TITLE
K8s: Node resource eviction thresholds and resource quotas (content migration)

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Jiras: DOC-1916 DOC-1659
Staged preview: https://docs.redis.com/staging/DOC-1916/kubernetes/re-clusters/node-resources/

### CODE REVIEW

File: content/kubernetes/re-clusters/node-resources.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+name:gke-7253cc19-42g0	MemoryPressure:False	DiskPressure:False
+```
+
+For more information about monitoring node conditions, see [Node conditions](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#node-conditions) on [kubernetes.io](https://kubernetes.io/docs/home/).
+
+## Resource quotas
+
+Kubernetes uses the `ResourceQuota` object to limit resource consumption per namespace. This lets you limit the number of objects created by a namespace, or the amount of compute resources consumed by a namespace. 
+
+The resource settings for Redis Enterprise for Kubernetes are defined in the `operator.yaml`.
```
Comment (yuvallevy2) [inline content/kubernetes/re-clusters/node-resources.md:44]: there are also resource settings inside the REC custom resources, https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md#redisenterpriseclusterspec

**Review by cmilesb — APPROVED** [review]

File: content/kubernetes/memory/node-resources.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+Title: Manage node resources
+linktitle: Manage resources
+description: Recommendations on managing node resources and setting evictions thresholds with Redis Enterprise for Kubernetes.
```
Comment (rrelledge) [inline content/kubernetes/memory/node-resources.md:4]:
```suggestion
description: Recommendations on managing node resources and setting eviction thresholds with Redis Enterprise for Kubernetes. 
```

File: content/kubernetes/memory/node-resources.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Node eviction thresholds
+
+ Kubernetes uses [node-pressure eviction](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/) to terminate pods and free up node resources. Redis Enterprise for Kubernetes works best when you use the recommendations below.
+
+Eviction thresholds are managed by [kubelet arguments](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/).
+
+Redis recommends setting a high [soft eviction threshold](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#soft-eviction-thresholds). This changes the node condition early enough to alert the administrator.
+
+We also recommend setting the [`eviction-soft-grace-period`](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#soft-eviction-thresholds) high enough for the administrator to scale the cluster.
```
Comment (rrelledge) [inline content/kubernetes/memory/node-resources.md:23]: Maybe the recommendations could be a bulleted list.

File: content/kubernetes/memory/node-resources.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+For more information about configuring eviction thresholds on specific platforms, see ["Freeing node resources"](https://docs.openshift.com/container-platform/4.9/nodes/nodes/nodes-nodes-garbage-collection.html) for OpenShift or ["Cluster architecture"](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture#node_allocatable) for GKE.
```
Comment (rrelledge) [inline content/kubernetes/memory/node-resources.md:25]:
```suggestion
For more information about configuring eviction thresholds on specific platforms, see [Freeing node resources](https://docs.openshift.com/container-platform/4.9/nodes/nodes/nodes-nodes-garbage-collection.html) for OpenShift or [Cluster architecture](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture#node_allocatable) for GKE.
```

File: content/kubernetes/memory/node-resources.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+name:gke-55d1ac88-213c	MemoryPressure:False	DiskPressure:False
+name:gke-55d1ac88-vrpp	MemoryPressure:False	DiskPressure:False
+name:gke-7253cc19-42g0	MemoryPressure:False	DiskPressure:False
+```
+
+For more information about monitoring node conditions, see [Node conditions](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#node-conditions) on [kubernetes.io](https://kubernetes.io/docs/home/).
+
+## Resource quotas
+
+Kubernetes uses the `ResourceQuota` object to limit resource consumption per namespace. This lets you limit the number of objects created by a namespace, or the amount of compute resources consumed by a namespace.
```
Comment (rrelledge) [inline content/kubernetes/memory/node-resources.md:42]:
```suggestion
Kubernetes uses the `ResourceQuota` object to limit resource consumption per namespace. This lets you limit the number of objects created by a namespace or the amount of compute resources consumed by a namespace. 
```

**Review by rrelledge — APPROVED** [review]
Looks good to me.

**Pull request merged**